tests/unit/s2n_client_supported_versions_extension_test.c

```diff
@@ -378,9 +377,6 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_stuffer_free(&extension));
     }
 
-    /* Since the supported_version extension replaces the version field
```

Comment:
Why did you delete this comment?

Reply:
I thought it was to be replaced with the quote from RFC. Do I keep both ?

Reply:
Keep both. The quote from the RFC describes your test. Not sure what this other comment is for but you didn't alter its test so you shouldn't change the comment.

Reply:
My bad. I deleted a comment for a different test. Putting it back.